app/models/hca_azul_client.rb

```diff
@@ -28,6 +28,14 @@ class HcaAzulClient
     'x-domain-id' => "#{ENV['HOSTNAME']}"
   }.freeze
 
+  # list of words that are common among Azul filter values that result in too many matches
+  # this can cause HTTP 413 Payload Too Large or HTTP 414 URI Too Long errors when executing
+  # search requests in Azul
+  COMMON_AZUL_TERMS = %w[cell cells human single-cell reveals sequencing rna-seq analysis atlas rna disease].freeze
```

Comment:
Not being able to query Azul for "human" seems like the only notable downside of this approach.

```suggestion
  COMMON_AZUL_TERMS = %w[cell cells human single-cell reveals sequencing rna-seq analysis atlas rna disease].freeze
```

Reply:
Agreed, now that I think about it.  Interestingly, I re-ran the analysis, and the results of common words I got was different.  It seems that these values are not static, which seems odd.  I'm going to hold off merging this until I get a better handle on what is going on here.

Reply:
Turns out we have an unrelated bug in creating these queries where we're clobbering earlier matches, and then there .  Which means this problem is actually _much_ worse than we thought.  I'm going to reassess this and see if there's more that needs to be done here.